_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -463,6 +463,11 @@ public function get_all_options() {
 					);
 					break;
 
+				case 'jetpack_event_tracking':
+					jetpack_require_lib( 'class.jetpack-user-event-tracking' );
```

Comment:
This is throwing a fatal, can't find the lib

Reply:
Oh, because of #9003

Reply:
Yes. While coding this today, I extracted the class to make the updates so it works with it. We will merge this after Enej and Miguel merge the other.